assets/queries/terraform/alicloud/oss_bucket_allows_delete_from_all_principals/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "id": "8c0695d8-2378-4cd6-8243-7fd5894fa574",
+    "queryName": "OSS Bucket Allows Delete Action From All Principals",
+    "severity": "HIGH",
+    "category": "Access Control",
+    "descriptionText": "OSS Bucket should not allow delete action from all principals.",
```

Comment:
```suggestion
 "descriptionText": "OSS Bucket should not allow delete action from all principals, as to prevent leaking private information to the entire internet or allow unauthorized data tampering/deletion. This means the 'Effect' must not be 'Allow' when the 'Action' is DeleteBucket, for all Principals.",
```